ext/EmulatorModelsExt.jl

```diff
@@ -60,4 +62,25 @@ function AA.N_activated_per_mode(
     end
 end
 
+"""
+    CalibratedAerosolActivationParameters(ekp_params)
+
+    - `ekp_params` - parameters from the trained Ensemble Kalman Process
+Returns a calibrated set of aerosol activation parameters
+"""
+function AA.CalibratedAerosolActivationParameters(
```

Comment:
Instead of creating a new function, could we extend this one: `AerosolActivationParameters` ? It's defined in [here](https://github.com/CliMA/CloudMicrophysics.jl/blob/26e29d49ae397b6d1039c78db7a8d88b98f0658c/src/parameters/AerosolActivation.jl#L40C10-L40C37)

Reply:
Just kidding, it does work! Yay for multiple dispatch.